oauth2client/client.py

```diff
@@ -469,6 +467,12 @@ def clean_headers(headers):
     return clean
 
 
+def _parse_expiry(expiry):
+    if expiry and isinstance(expiry, datetime.datetime):
+        return expiry.strftime(EXPIRY_FORMAT)
+    return expiry
```

Comment:
I just realized this is bad if `expiry` is False-y but not `None`. Can you move the final `return expiry` since we really just want `None` to be returned (which is done by default)?